### PR TITLE
Fix  automatic execute method SubmitPrint

### PR DESCRIPTION
**Pull request by chuongmep**
# Summary of the Pull Request
This is a problem when user snoop to PrintManager, method **SubmitPrint** invoke
**What is this about:** 
Revit automatic print a file pdf active window
**Description:** 
image

## Quality Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: RevitLookup/Core/Streams/ElementMethodsStream.cs [hunk 1]
```diff
@@ -30,6 +30,7 @@ public void Stream(Type type)
 
         foreach (var methodInfo in methods)
         {
+            if(IsMethodExcept(methodInfo)) continue;
```
Comment (Nice3point) [inline RevitLookup/Core/Streams/ElementMethodsStream.cs:33]: Open `RevitLookup/Core/CollectorExtensions/DataFactory.cs`

RollBack your changes and add the following line

```c#
if (declaringType == typeof(PrintManager) && info.Name == nameof(PrintManager.SubmitPrint))
    return null;
```

**Review by Nice3point — CHANGES_REQUESTED** [review]

File: RevitLookup/Core/Streams/ElementMethodsStream.cs (hunk 1 above)
Comment (chuongmep) [reply RevitLookup/Core/Streams/ElementMethodsStream.cs]: i think it ok, I will change it to return value true and change code base.

File: RevitLookup/Core/CollectorExtensions/DataFactory.cs [hunk 2]
```diff
@@ -105,7 +105,8 @@ public Data Create(MethodInfo methodInfo)
 
         if (info.GetParameters().Length > 0 || info.ReturnType == typeof(void))
             return null;
-
+        if (declaringType == typeof(PrintManager) && info.Name == nameof(PrintManager.SubmitPrint))
+            return new BoolData("SubmitPrint",true);
```
Comment (Nice3point) [inline RevitLookup/Core/CollectorExtensions/DataFactory.cs:109]: use it `return new BoolData(nameof(PrintManager.SubmitPrint), false);`

a space after the comma, and perhaps it's better to display false, the print job was not sent

**Review by Nice3point — CHANGES_REQUESTED** [review]

**Comment by Nice3point** [comment]
but in other things, I'll fix it myself, thanks for the PR)

**Pull request merged**